loops-and-arrays.js

```diff
@@ -31,6 +35,18 @@ const numbers = [2, 4, 5, 29, 38];
 
 
 
+const numbers = [2, 4, 5, 29, 38];
+console.log(numbers);
+
+for(let i = 0; i < numbers.length; i++) {
+    if(numbers[i]%2==0)
```

Comment:
Volgens de aandachtspunten peer review is het aangeraden de spaties tussen de entries en de operatoren te gebruiken :
if (i % 2 === 0) {
  }

Ook was het tijdens de les aangeraden om vergelijkingsoperator === te gebruiken in plaats van ==.
Vergelijkingsoperator === vergelijkt op waarde en op type, == vergelijkt alleen op waarde.